packages/generators/add-generator.ts

```diff
@@ -116,377 +115,401 @@ export default class AddGenerator extends Generator {
 		// eslint-disable-next-line
 		const isDeepProp: any[] = [false, false];
 
-		return this.prompt([
-			AutoComplete("actionType", "What property do you want to add to?", {
-				pageSize: 7,
-				source: searchProps,
-				suggestOnly: false
-			})
-		])
-			.then(
-				(actionTypeAnswer: { actionType: string }): void => {
-					// Set initial prop, like devtool
-					this.configuration.config.webpackOptions[actionTypeAnswer.actionType] = null;
-					// update the action variable, we're using it later
-					action = actionTypeAnswer.actionType;
+		const actionTypeAnswer: {
+			actionType: string,
+		} =  await this.prompt([
+			AutoComplete(
+				"actionType",
+				"What property do you want to add to?",
+				{
+					pageSize: 7,
+					source: searchProps,
+					suggestOnly: false,
+				},
+			),
+		]);
+
+		// Set initial prop, like devtool
+		this.configuration.config.webpackOptions[
+			actionTypeAnswer.actionType
+		] = null;
+		// update the action variable, we're using it later
+		action = actionTypeAnswer.actionType;
+
+		if (action === "entry") {
+			const entryTypeAnswer: {
+				entryType: boolean,
+			} = await this.prompt([
+				Confirm("entryType", "Will your application have multiple bundles?", false),
+			]);
+
+			// Ask different questions for entry points
+			const entryOptions = await entryQuestions(self, entryTypeAnswer);
+
+			this.configuration.config.webpackOptions.entry = entryOptions;
+			this.configuration.config.item = action;
+
+		} else {
+			if (action === "topScope") {
+
+				const topScopeAnswer: {
+					topScope: string,
+				} = await this.prompt([
+					Input("topScope", "What do you want to add to topScope?"),
+				]);
+
+				this.configuration.config.topScope.push(topScopeAnswer.topScope);
+				return;
+			}
+		}
+
+		const temp: string = action;
```

Comment:
reassigning a const

Reply:
Where? temp is never reassigned anywhere and here it is declared.